src/commands/new-proj.ts

```diff
@@ -190,6 +199,88 @@ async function initQuarkusProject() {
   }
 }
 
+/**
+ * Attachs AWS to the project
+ * 
+ * @param {any} instance Vorpal instance for inquiry.
+ */
+async function attachAWS(instance: any) {
```

Comment:
What is the purpose of this? It seems to do quite lot of thinks and most of them do not make sense